server/protocols/jmap/src/test/java/org/apache/james/jmap/model/MessageFactoryTest.java

```diff
@@ -213,7 +219,13 @@ public void headersShouldBeUnfoldedAndDecoded() throws Exception {
             .preview("(Empty)")
             .textBody(Optional.of(""))
             .htmlBody(Optional.empty())
+            .isAnswered(false)
```

Comment:
Please reject the use of **is\*** and **keyword** properties on the same builder...